exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,39 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative() { 
```

Comment:
If You want to make your life easier, please remember about indentation.
Please check this:
https://courses.cs.washington.edu/courses/cse154/17au/styleguide/js/spacing-indentation-js.html#:~:text=Spacing%20and%20indentation%20should%20be,than%20the%20previous%20line%27s%20indentation.